src/auth.rs

```diff
@@ -49,7 +49,7 @@ impl Auth {
     pub fn supports_netrc(auth_type: AuthType) -> bool {
         match auth_type {
             AuthType::basic => true,
-            AuthType::bearer => false,
+            AuthType::bearer => true,
```

Comment:
If you're interested in more strict compatibility, this could be changed to be aware of `httpie_compat_mode`? Let me know if you'd prefer being stricter like that, I think that's just a few lines more change.

Reply:
I don't think that will be necessary. Can you remove `supports_netrc` since all auth types can read the `.netrc` file?

Reply:
Sure—I was leaving this in case of future AuthTypes ever being added that might not support it.